experiment_scripts/omniglot_maml-omniglot_1_8_0.1_64_5_0_few_shot.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/sh
 
-export GPU_ID=$1
+export GPU_ID=1
```

Comment:
This modification is wrong. What you want is $1, which passes the second argument entered into the terminal after the script name and sets it as the GPU idx that the program should be using.

Reply:
If you agree to this change, please modify the file and I'll accept your PR.

Reply:
Sure, I have modified it. Yes, it makes sense to keep it to $1. Thanks